.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/Tribler/py-ipv8.git
 [submodule "Tribler/anydex"]
 	path = Tribler/anydex
-	url = https://github.com/Tribler/anydex-core.git
```

Comment:
This should be the main Tribler repo

Reply:
I agree, but unfortunately https://github.com/Tribler/anydex-core/pull/30 still needs to be merged.

Reply:
I have created a separate `asyncio` [branch](https://github.com/Tribler/anydex-core/tree/asyncio) on the upstream repo, so this one can point to that branch, without changing the submodule URL 👍

Reply:
The PR was already pointing to a pre-existing asyncio branch as far as I know, but thanks for merging!